web/src/context/Stats19.svelte

```diff
@@ -0,0 +1,234 @@
+<script lang="ts">
+  import type { ExpressionSpecification } from "maplibre-gl";
+  import {
+    CircleLayer,
+    VectorTileSource,
+    type LayerClickInfo,
+  } from "svelte-maplibre";
+  import { QualitativeLegend } from "svelte-utils";
+  import { makeRamp, Popup } from "svelte-utils/map";
+  import { HelpButton, layerId } from "../common";
+  import { assetUrl } from "../stores";
+
+  let show = false;
+  let state = {
+    pedestrians: true,
+    cyclists: true,
+    minYear: 2017,
+    maxYear: 2023,
+  };
+
+  function makeFilter(_: any): ExpressionSpecification {
+    let includeTypes: ExpressionSpecification = ["any"];
+    if (state.pedestrians) {
+      includeTypes.push(["get", "pedestrian"]);
```

Comment:
The data we produce also has `horse_rider` and `other`, leftovers from what's needed for the original use case. When we start using this to match to neighbourhoods and count, we will also need to exclude. Maybe we should exclude upfront and be simpler...

Reply:
Exclude as part of the data_prep process you mean? That seems reasonable.

Reply:
Low energy now, but I'll take it as a TODO.